content/topics/github/intro/_index.md

```diff
@@ -19,29 +19,27 @@ Please go through this content and do your best to understand all the things.
 
 ## Github versus Git
 
-One thing to know is that Git and Github are not the same thing. Git is a foundational tool, you don't need to use Github if you want to use Git.
+One thing to know is that Git and Github are not the same things. Git is a foundational tool; you don't need to use Github if you want to use Git.
```

Comment:
```suggestion
One thing to know is that Git and Github are not the same thing. Git is a foundational tool; you don't need to use Github if you want to use Git.
```

Reply:
Jacob, Grammarly says things, not thing. It will throw an error if I change it back